addons/website/static/src/components/configurator/configurator.js

```diff
@@ -110,6 +113,34 @@ class DescriptionScreen extends Component {
         }
     }
 
+    clearIndustrySelection() {
+        this.industrySelection.el.value = '';
+        this.industrySelection.el.parentNode.dataset.value = '';
+        this.dispatch('selectIndustry', undefined, undefined);
+    }
+
+    setSelectedIndustry(label) {
+        this.industrySelection.el.parentNode.dataset.value = label;
+        const id = this.labelToId[label];
+        this.dispatch('selectIndustry', label, id);
+        this.checkDescriptionCompletion();
+    }
+
+    customizeMenu(_) {
+        const items = $(".ui-menu a");
+        if (!this.autocompleteHasResults) {
+            items.addClass("no-result");
+        } else {
+            items.removeClass("no-result");
+        }
+    }
+
+    disableKeyboardNav(ev) {
+        if (!this.autocompleteHasResults) {
+            ev.preventDefault();
+        }
+    }
```

Comment:
All of this needs docs and comments. You don't code for you, you code for everyone which will ever look at this code. And not everything is obvious here: for example, just by seeing this: why is there a method call `customizeMenu` (very generic name) which seems to only be there to handle a "no-result" class? Or is it a function override? And why does it receive an unnamed and unused parameter? Also is that class a library class or should it be prefixed by o_ to indicate it is one of our class?